frame/oracle/src/lib.rs

```diff
@@ -790,10 +831,13 @@ pub mod pallet {
 			let mut to32 = AccountId32::as_ref(&account);
 			let address: T::AccountId = T::AccountId::decode(&mut to32).unwrap_or_default();
 
+			if prices.len() as u32 >= Self::asset_info(price_id).max_answers {
+				return Err("Max answers reached")
```

Comment:
how this error is handled?

Reply:
the error is generated off chain in an offchain worker, the result of no error is the node prepping and sending in a tx, the error is logged and results in no tx being prepped

Reply:
so you say that if error never handled neither logged it it still saved somewhere ?

![image](https://user-images.githubusercontent.com/757125/144762282-9642c704-972c-4086-b7b9-33de133326eb.png)

Reply:
ahh ya I guess it wont added info loggings